.travis.yml

```diff
@@ -1,5 +1,9 @@
+sudo: false
```

Comment:
This tells travis that sudo permissions are explicitly prohibited. It's more secure (and certainly a best practice for [older repos](https://github.com/sindresorhus/sudo-block/blob/master/.travis.yml)). Sudo permissions were actually granted by default for builds before 2015 (which r-dom is likely still classified as).

You can read even more [here](https://docs.travis-ci.com/user/ci-environment) if you wish. Ctrl+F for "sudo"